media/MetadataViewer/index.css

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+body {
+  background-color: #1E1E1E;
+  max-width: 800px;
+  margin-left: 20px;
+}
+
+/* white테마일 때 처리 */
+body.vscode-light {
+  background-color: #ececec;
+}
+
+body.vscode-light .view-item-header{
+  color: #252526;
```

Comment:
This is my personal preference, but what about setting the color as same as '--vscode-editor-foreground(#d4d4d4)' ?

####  Reference
![0928-vscode-foreground](https://user-images.githubusercontent.com/17171963/192768565-9adfa973-f02a-479e-a7e7-c17420eada2b.png)

Reply:
![Vscode Settings](https://user-images.githubusercontent.com/17171963/192770479-4ab314ec-c820-4189-9848-3173f97ab9a4.png)

Reply:
I'll apply your suggestion.